main/swadge_esp32.c

```diff
@@ -996,10 +1008,15 @@ void cleanupOnExit(void)
  */
 void switchToSwadgeMode(swadgeMode* mode)
 {
-#if !defined(MONKEY_AROUND)
-    pendingSwadgeMode = mode;
-    isSandboxMode = false;
+#if defined(EMU)
+    if (!monkeyAround || lockMode)
+#else
```

Comment:
You don't need the `#else` part here. You can have `{...}` without a logical check before it.